lib/renderer.js

```diff
@@ -390,12 +393,12 @@
                     bounds = childBounds;
                 } else {
                     // Compute containing rect of the union of the bounds and maskBounds
-                    bounds.union(childBounds);
+                    bounds = bounds.union(childBounds);
```

Comment:
We were computing the union and throwing it away before. This fixes layer comp / document clipping!